### PR TITLE
Skipping test "[sig-devex][Feature:ImageEcosystem][perl][Slow] hot deploy for openshift perl image hot deploy should work" for ARM64

### PR DESCRIPTION
**Pull request by deepsm007**
Skipping the Hot Deploy perl image test for ARM64

https://github.com/openshift/ocp-on-arm/issues/3

/cc @gabemontero 
/cc  @yselkowitz

### CODE REVIEW

**Comment by gabemontero** [comment]
a draft PR will result in no e2e's getting submitted @deepsm007 

go ahead and create a normal PR

also, for the change you made, you may need to run `make update` as well

**Comment by deepsm007** [comment]
> a draft PR will result in no e2e's getting submitted @deepsm007
> 
> go ahead and create a normal PR
> 
> also, for the change you made, you may need to run `make update` as well

Yes, I did ran `make update` before pushing the commit

File: test/extended/image_ecosystem/s2i_perl.go
Comment (gabemontero) [inline test/extended/image_ecosystem/s2i_perl.go:25]: this is too high up ... this will end up skipping more tests than you expect

Comment (gabemontero) [reply test/extended/image_ecosystem/s2i_perl.go:25]: move this below line 

g.It("should work", func() {

**Review by gabemontero — CHANGES_REQUESTED** [review]

**Comment by gabemontero** [comment]
/approve
/lgtm
/hold

I put the hold so between you and I @deepsm007 we can confirm the test still runs on x86

once one of us does that the hold can be cancelled

in theory you should have permission to cancel the hold

File: test/extended/image_ecosystem/s2i_perl.go
Comment (yselkowitz) [inline test/extended/image_ecosystem/s2i_perl.go:51]: NodeOSArchIs does not work as you would expect, see #26435 for example.  This needs to be reworked along those lines.
/hold

**Review by yselkowitz — CHANGES_REQUESTED** [review]

File: test/extended/image_ecosystem/s2i_perl.go
Comment (yselkowitz) [inline test/extended/image_ecosystem/s2i_perl.go:21]: Rename appropriately, e.g. archHasModPerl

File: test/extended/image_ecosystem/s2i_perl.go
Comment (yselkowitz) [inline test/extended/image_ecosystem/s2i_perl.go:70]: Ditto.

**Review by yselkowitz — CHANGES_REQUESTED** [review]

File: test/extended/image_ecosystem/s2i_perl.go
Comment (yselkowitz) [inline test/extended/image_ecosystem/s2i_perl.go:71]: `g.Skip("mod_perl based builder image is not available on arm64")`

File: test/extended/image_ecosystem/s2i_perl.go
Comment (yselkowitz) [inline test/extended/image_ecosystem/s2i_perl.go:16]: not necessary with `g.Skip`

**Review by yselkowitz — CHANGES_REQUESTED** [review]

File: test/extended/image_ecosystem/s2i_perl.go
Comment (deepsm007) [reply test/extended/image_ecosystem/s2i_perl.go:16]: IIUC e2eskipper is not used anymore?

Comment (deepsm007) [reply test/extended/image_ecosystem/s2i_perl.go:16]: Ginko driver should skip the test

**Comment by gabemontero** [comment]
I'm fine with the location of the skip as it currently exists

I'm deferring to @yselkowitz to work with @deepsm007 as he is already doing to get the mechanics right, and he can lgtm when he is happy

I'm confident where that lands will be fine, but as I'm still asking for the hold to stay until we confirm the test still runs for x86, I can make any last minute asks.  Otherwise we'll release the hold when all is well

thanks gentlemen

**Comment by gabemontero** [comment]
ok the skipped test for arm ran on x86:

`


: [sig-devex][Feature:ImageEcosystem][perl][Slow] hot deploy for openshift perl image  hot deploy test should work | 2m44s
-- | --


`

from https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/26507/pull-ci-openshift-origin-master-e2e-gcp-image-ecosystem/1445464297516830720

/hold cancel

**Pull request merged**
